.github/workflows/ci.yml

```
@@ -42,6 +42,32 @@ jobs:
+            jinja2 "$f" values.yml > "$out"
+            rm -f $f
+          done
+
+      - name: Render all {{ }} folders
+        run: |
+          # Find all directories with {{ ... }} in their names
+          find . -depth -type d -name '*{{*}}*' | while IFS= read -r dir; do
+              echo "Found folder $dir"
+              # Render folder name with jinja-cli using values.yml
+              new_dir=$(jinja2 -f values.yml <<< "$dir")
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)